### DISCUSSION TITLE
Built with taffy

### DISCUSSION THREAD

**Discussion by paritosh-gupta** [DioxusLabs/taffy#237] [Show and tell]
Hi ya'll.

I'm currently building [CopyCat.dev](https://copycat.dev). Still quite early beta i'd say.  Most of the tech is in rust and I used taffy to build part of the layout engine which helps us guess the flex properties to use when layouting elements on screen.  Still a lot of work to be done to get it working right on my end but just wanted to share and thank ya'll. Hope to contribute back in the near future  🙂

**Comment by alice-i-cecile** [comment]
Thanks for letting us know :D That's super cool; please feel welcome to pitch in upstream when things come up! I've mostly been focused on Bevy's use cases, and diverse perspectives are very valuable.
